src/benchmarks/micro/libraries/System.Text.Json/Serializer/WriteJson.cs

```diff
@@ -67,5 +82,42 @@ public async Task SerializeToStream()
 
         [Benchmark]
         public string SerializeObjectProperty() => JsonSerializer.Serialize(_objectWithObjectProperty);
+
+        private sealed class ArrayBufferWriter : IBufferWriter<byte>
+        {
+            private int _offset = 0;
+            private byte[] _buffer = new byte[128];
```

Comment:
nit: if there is any particular reason why this value is set to 128 (like "this is what we do in BCL") it would be nice to add a comment here.

Reply:
No real reason, it just anticipates the expected output size of this specific benchmark. It doesn't have any real bearing on the benchmark itself, since the buffer will resize to the required size on the first run.